### DISCUSSION TITLE
Glitch in network grāph

### DISCUSSION THREAD

**Discussion by Dark0017** [community/community#11967] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/51037059/155331288-a8d61e25-b336-41a2-892b-0f452a92737a.png)
Help, why. What do

**Comment by pzzd** [comment]
I noticed this too. The canvas for the network graph is too short. This affects Safari and Chrome.

**Reply by harshraj8843** [reply]
> This affects Safari and Chrome.

`Brave browser` and `firefox` too

https://github.community/t/network-graph-not-visible/231903/

**Reply by akinomyoga** [reply]
This issue was also reported at https://github.com/github/feedback/discussions/12061 where the problem is explained.

**Reply by pzzd** [reply]
Looks like it's fixed now!

**Reply by harshraj8843** [reply]
> Looks like it's fixed now!

For me too

**Comment by jupmorenor** [comment, marked as answer]
Changing zoom can make it show a little more, anyway cursor interaction and commit popups are buggy,

**Reply by yan12125** [reply]
A good hint! Everything, including dragging and commit popus, seems fine for me after I changed the zoom level from 133% (my browser default) to 100% (original size). Tried Firefox & Chromium on Linux.

**Comment by rollanda21** [comment]
@jupmorenor  I changed zoom, resized the chrome window, and changed the canvas height on the navigator side but none of this works.
